nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/web/common/requestcontroller/filter/NuxeoRequestControllerFilter.java

```diff
@@ -308,7 +310,9 @@ private static DateFormat httpExpiresDateFormat() {
 
     /**
      * Set cache parameters to httpResponse.
+     * @deprecated Since 10.3
```

Comment:
Not sure we need these changes on `9.10`?

Reply:
If it simplifies the backport we keep the same code.

Reply:
That's my point, we don't keep the same code, no `addCacheHeader` for instance.

Reply:
Ah yes. You're right these `@deprecated` aren't needed in the backport given we've fixed up the backport to not change this method.

Reply:
was just a heads' up really since it's gone in 10.3. removed 👍